query.js

```diff
@@ -26,4 +26,18 @@ const allCourseCategory = `
 }
 `;
 
-module.exports = { allCourseQuery, allCourseCategory };
+const allProjectArticle = `
+{
+  allContentfulProgetti {
+    nodes {
+      id
+      titolo
+      project_category {
+        slug
+      }
+    }
+  }
+}
```

Comment:
Andrei a fare la query in questo modo 
```
{
  allContentfulProgetti {
    nodes {
      id
      titolo
      project_category {
        slug
      }
      body {
        body
      }
    }
  }
}
```

E poi in node andrei a controllare che questo body non sia `null` e solo a quel punto creerei la pagina del singolo articolo